src/docs/exampleComponents/LineChart/LineChartNegativeValuesWithReferenceLines.js

```diff
@@ -0,0 +1,132 @@
+import React, { PureComponent } from 'react';
+import { CartesianGrid, Line, LineChart, ReferenceLine, ResponsiveContainer, XAxis, YAxis } from 'recharts';
+
+const data = [
+  {
+    x: -50,
+    y: -50,
+  },
+  {
+    x: 0,
+    y: 0,
+  },
+  {
+    x: 50,
+    y: 50,
+  },
+  {
+    x: 100,
+    y: 100,
+  },
+  {
+    x: 150,
+    y: 150,
+  },
+  {
+    x: 200,
+    y: 200,
+  },
+  {
+    x: 250,
+    y: 250,
+  },
+  {
+    x: 350,
+    y: 350,
+  },
+  {
+    x: 400,
+    y: 400,
+  },
+  {
+    x: 450,
+    y: 450,
+  },
+  {
+    x: 500,
+    y: 500,
+  },
+];
+
+const minX = Math.min(...data.map((d) => d.x));
+const minY = Math.min(...data.map((d) => d.y));
+
+export default class Example extends PureComponent {
+  static demoUrl = 'https://stackblitz.com/edit/react-ts-ltwtbv?file=Chart.tsx';
```

Comment:
since we use codesandbox for our other examples can we use codesandbox here?

Reply:
No problem! Is it ok if it's TypeScript?

Reply:
Yep, perfect

Reply:
Done 😄